tensorflow/core/kernels/pooling_ops_common.h

```diff
@@ -68,6 +68,69 @@ struct PoolParameters {
   TensorFormat data_format;
 };
 
+#ifdef TENSORFLOW_USE_SYCL
```

Comment:
This change set might cause conflict as it appears in two pull requests. ( https://github.com/lukeiwanski/tensorflow/pull/99 )